content/includes/dockerfile-labs-channel.md

```diff
@@ -1,7 +0,0 @@
----
----
-
-{{% experimental %}}
-The "labs" channel provides early access to Dockerfile features that are not
-yet available in the stable channel.
```

Comment:
![image](https://github.com/user-attachments/assets/fab11017-908e-445b-9125-ebfc88422944)

Does not give much context while previously we got this sentence. Can we keep that?